### PR TITLE
VideoPath setter unification for local/remote path.

### PR DESCRIPTION
**Pull request by ghost**
Thank you for your contribution to MagicScript Project.
Before submitting this PR, please make sure:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

### CODE REVIEW

File: src/platform/lumin-runtime/elements/builders/video-builder.js [hunk 1]
```diff
@@ -42,9 +43,11 @@ export class VideoBuilder extends QuadBuilder {
...
 
         if (videoPath !== undefined) {
-            this._callNodeAction(element, 'setVideoPath', videoPath);
-        } else {
-            this._callNodeAction(element, 'setVideoUri', videoUri);
+          if (isUrl(videoPath)) {
+            this._callNodeAction(element, "setVideoUri", videoPath);
+          } else {
+            this._callNodeAction(element, "setVideoPath", videoPath);
+          }
```
Comment (pleszkiewicz) [inline src/platform/lumin-runtime/elements/builders/video-builder.js:50]: I think it's not enough to update `videoUri` in **create** function only. What about **update** function? Can we get rid of `videoPath` and `videoUri` and support only one property (visible from JSX)?

Comment (grozdanov) [reply src/platform/lumin-runtime/elements/builders/video-builder.js]: @SPaluchiewiczSofomo and @pleszkiewicz, please review the refactored code.

**Review by pleszkiewicz — APPROVED** [review]
Code: 👍

**Review by grozdanov — APPROVED** [review]
👍 Thank you @SPaluchiewiczSofomo !

**Pull request merged**